examples/mbr_sweep_example.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (C) 2023 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-FileCopyrightText: 2024 ANSYS, Inc. All rights reserved
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""
+.. _multi_blade_row_example:
+
+Multi blade row meshing example
+-------------------------------
+This basic example shows how to set up a multi blade row meshing instance and execute it in parallel.
+
+"""
+
+#################################################################################
+# Perform required imports
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Perform the required imports. It is assumed that the ``ansys-turbogrid-core``
+# package has been installed.
+
+
+import pathlib
+import time
+
+install_path = pathlib.PurePath(__file__).parent.parent.as_posix()
+
+from ansys.turbogrid.core.multi_blade_row.multi_blade_row import multi_blade_row as MBR
+
+
+all_face_areas = {}
+all_element_counts = {}
+
+
+#################################################################################
+# Create a multi_blade_row and initialize it
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
@Acohen-work Make sure you use the period at the end of the sentences consistently. Also make sure that the underlining is the same length as the heading text.